front/app/containers/Admin/projects/project/participationContext/components/inputs/IdeationInputs/index.tsx

```diff
@@ -110,6 +110,17 @@ export default ({
   const hasAnonymousParticipationEnabled = useFeatureFlag({
     name: 'anonymous_participation',
   });
+
```

Comment:
Used useRef so that even if the fields are changed to the defaults and saved then the config fields will still show. Only when leaving the config and returning will they disappear.

Reply:
Sounds good to me :+1: